src/domain/signup/permissions.ts

```diff
@@ -33,7 +42,10 @@ export const checkCanUserDoSignupAction = ({
     case SIGNUP_ACTIONS.SEND_MESSAGE:
```

Comment:
A public note to myself: In the back-end, sending a message is allowed even if the admin user is not the one who created the registration. Since the admin isn't able to view the signups if they aren't the registration creator, I should probably change this to be the same way in the back-end.

Reply:
This was addressed in https://github.com/City-of-Helsinki/linkedevents/pull/805